src/_zkapauthorizer/resource.py

```diff
@@ -648,3 +711,33 @@ def bad_request(reason="Bad Request"):
         b"Bad Request",
         reason.encode("utf-8"),
     )
+
+
+async def recover(
```

Comment:
I added this not-extremely-useful client helper API because 3 or 4 tests were essentially duplicating it.